DTS-STN/canadian-dental-care-plan
# AB#2998 - Fix communication method email not found

### Description


Fix communication method email not found. This can be reproduced on DEV.

### Related Azure Boards Work Items


[AB#2998](https://dev.azure.com/DTS-STN/1fc40a8f-28cf-47bc-b6e4-1c234bd06177/_workitems/edit/2998)

### Screenshots (if applicable)


### Checklist


### Test Instructions


Start the apply flow and go to the `[apply|intake]/:id/communication-preference` and the page should load.

### Additional Notes
